lib/global.gd

```diff
@@ -61,7 +61,7 @@ DeclareGlobalFunction("IsValidIdentifier");
 ##  </Description>
 ##  </ManSection>
 ##
-DeclareGlobalFunction("ValueGlobal");
+DeclareSynonym("ValueGlobal", VALUE_GLOBAL);
```

Comment:
I agree with @stevelinton : I'd leave this declaration here, and then just change the `.gi` file to `InstallGlobalFunction( ValueGlobal, VALUE_GLOBAL )`, simply for symmetry with the rest.

The difference is in the function names:
```
gap> DeclareSynonym("foo", VALUE_GLOBAL);
gap> DeclareGlobalFunction("bar");
gap> InstallGlobalFunction( bar, VALUE_GLOBAL );
gap> NameFunction(foo);
"VAL_GVAR"
gap> NameFunction(bar);
"bar"
```

Which reminds me of another thing: `VALUE_GLOBAL` itself is just an alias for `VALUE_GVAR`, and it is not even write protected: You can e.g. do `VALUE_GLOBAL := 1`...

So we may want to unify that, too. But of course we have to be careful because there are packages using almost any of the three names in provided for each of the functions dealing with globals *sigh*. (Strictly speaking, packages using the undocumented functions `VALUE_GLOBAL`, `VALUE_GVAR` etc are doing something wrong, and we are not responsible for keeping them working, but, well, let's be realistic, people will still yell at us, not the package authors, if we change this without either providing backward compatibility code, or else getting all packages using it fixed first....